src/Core/Checkout/Document/Renderer/RenderedDocument.php

```diff
@@ -86,4 +95,12 @@ public function getConfig(): array
     {
         return $this->config;
     }
+
+    /**
+     * @param array<string, string> $files
+     */
+    public function setFiles(array $files): void
+    {
+        $this->files = $files;
```

Comment:
Set content with formats array{pdf: 'content with pdf', html: 'content with html'}

Reply:
Hi @nguyenytran 

Instead of adding new setFiles(array $files) and getHtml (string $fileType = FileTypes::PDF). 

We can simply and straightforward code by:
 
```
  public function __construct(
        private readonly string $html = '',
        private readonly string $htmlAccessibility = '',
  ....
  public function getHtmlAccessibility(): string
    {
        return $this->htmlAccessibility;
    }
```

In this way, we can easily realize that:

There are, methods supporting HTML and htmlAccessibility without going through filetype
 
CC: @silverDuy